src/Domain/Python/JacobianDiagnostic.cpp

```diff
@@ -0,0 +1,42 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Domain/Python/JacobianDiagnostic.hpp"
+
+#include <cstddef>
+#include <pybind11/pybind11.h>
+#include <string>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/JacobianDiagnostic.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Utilities/Gsl.hpp"
+
+namespace py = pybind11;
+
+namespace domain::py_bindings {
+
+namespace {
+template <size_t Dim, typename TargetFrame>
+void bind_jacobian_diagnostic_impl(py::module& m) {  // NOLINT
+  m.def(
+      "jacobian_diagnostic",
+      static_cast<tnsr::i<DataVector, Dim, typename Frame::ElementLogical> (*)(
+          const ::Jacobian<DataVector, Dim, Frame::ElementLogical,
+                           TargetFrame>&,
+          const tnsr::I<DataVector, Dim, TargetFrame>&, const ::Mesh<Dim>&)>(
+          &::domain::jacobian_diagnostic<Dim, TargetFrame>),
+      py::arg("jacobian"), py::arg("mapped_coords"), py::arg("mesh"));
```

Comment:
Name the second arg `TargetFrame == Frame::Inertial ? "inertial_coords" : "grid_coords"`. Then the py script will get the inertial coords from the volume data file by default, without having to specify the dataset name.

Reply:
Cool! I didn't know these were automagically available.

Reply:
This doesn't compile @nilsvu ... I get this error:

```
/Users/geoffrey/Codes/spectre/spectre/src/Domain/Python/JacobianDiagnostic.cpp:31:15: error: 'TargetFrame' does not refer to a value
      py::arg(TargetFrame == Frame::Inertial ? "inertial_coords"
              ^
/Users/geoffrey/Codes/spectre/spectre/src/Domain/Python/JacobianDiagnostic.cpp:21:32: note: declared here
template <size_t Dim, typename TargetFrame>
                               ^
/Users/geoffrey/Codes/spectre/spectre/src/Domain/Python/JacobianDiagnostic.cpp:31:46: error: expected '(' for function-style cast or type construction
      py::arg(TargetFrame == Frame::Inertial ? "inertial_coords"
```

Is there a way to do this with `std::is_same`? I tried but didn't get anything to build

Reply:
Right, frames are structs and no enums. std::is_same_v should work.